### DISCUSSION TITLE
Themes New canvas free to use

### DISCUSSION THREAD

**Discussion by Onslau** [mathoudebine/turing-smart-screen-python#217] [Themes]
![Gaming0_generated](https://user-images.githubusercontent.com/132712795/236567608-d6a412be-e629-459f-b37a-fdb299f46a04.jpg)

i want to make a landscape Theme but i dont know which size and format need help , i started with goverlay but this crap broke after 6 months , your piece of tech feels great and its easy to use buyed it today and iam happy !

this wallpaper is selfmade and a blanket its a work in progress!

if someone has an idea to this just use it !

**Comment by Onslau** [comment]
![Crycis3-wallpaper-V1-by-Designfjotten](https://user-images.githubusercontent.com/132712795/236568194-040eee7e-2ec1-4541-b27b-0f443e4c16da.jpg)

**Comment by Onslau** [comment]
![8-cyber-wallpaper-on-wallpapersafari-4k-wallpaper-technology](https://user-images.githubusercontent.com/132712795/236568391-1f0b2d8b-0a16-43e0-99ef-c96aebbac991.png)

**Comment by Onslau** [comment]
![HH](https://user-images.githubusercontent.com/132712795/236625175-3ab15e3b-8418-47f7-821a-79a7f347de6f.jpg)
have fun creating something

**Comment by Onslau** [comment]
![ez panel](https://user-images.githubusercontent.com/132712795/236680036-0875aec5-9eb6-4d7d-bf42-ae56d65add3a.jpg)

**Comment by Onslau** [comment]
![ROG EXTREME CANVAS](https://user-images.githubusercontent.com/132712795/236680052-7717ffce-4064-4d33-b915-c77c7a09d610.jpg)

**Comment by Onslau** [comment]
![Topo Goverlay Profile](https://user-images.githubusercontent.com/132712795/236680064-214e7427-faac-4d70-ac14-06cab0a5ff21.png)

**Comment by Onslau** [comment]
![rog 3](https://user-images.githubusercontent.com/132712795/236680880-97b1f328-efd4-414a-af80-5dbeb69fe059.jpg)

**Comment by Onslau** [comment]
![ROG EXTREME CANVAS 1](https://user-images.githubusercontent.com/132712795/236680896-3994dfd9-0881-4476-af1d-939fc2b49b43.jpg)

**Comment by Onslau** [comment]
[RED ROG 2.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11415199/RED.ROG.2.zip)

**Comment by mathoudebine** [comment]
Hey @onsau 
Thanks for sharing your work!
For the Turing smart screen, the background pictures must be 480x320 for landscape themes and 320x480 for portrait theme
I love your work!

**Reply by Onslau** [reply]
nah thanks i love your display and software i wish you can do more status bars circles gauges etc to experiment  with it i do overlays all day for 10 yrs

**Reply by Onslau** [reply]
and some tip , what ur tool needs is a snap function , i mean for cpu gpu number gauges etc to snap in in exact height etc with mouse and numbers it takes long to perfect it xD 

Your screen is great it need more gauges fonts for text and ui stuff and a snap in to make it fit perfect !
if i have more time i do more overlays and stuff !

**Reply by Paradoxum** [reply]
maybe arrow buttons that move the element around 1px at a time

**Comment by Onslau** [comment]
![Unbenannt](https://user-images.githubusercontent.com/132712795/236702869-8c94a0e2-48bd-4249-a067-f6c246c23fa6.png)
SHRINK IT !!!  cyberdemon overlay its 4k if someone compress it to the smartscreensize without loos please post it back thanks and have fun !

**Comment by Onslau** [comment]
![ironman ui](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/d267a825-375a-453f-b8e4-b0d754de2216)
Ironman ui shrink to use  pictures must be 480x320 for landscape themes and 320x480 for portrait theme

**Comment by Onslau** [comment]
![ironman ui CPU GPU](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/cd84caa8-0189-4955-8ba0-8d7649b3ba3c)
![ironman ui joker](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/282f9110-ddb1-48af-a519-23812b274182)
![ironman ui red](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/3f80ce2e-43a9-49aa-810f-c41d765f1a06)

**Comment by Onslau** [comment]
![ironman ui joker](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/15e29357-8d2e-47ec-b803-b26a97e37b8a)

**Comment by Onslau** [comment]
![ROBMENTUN](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/f1f4e20d-c93d-4f92-959c-2308e150f941)

**Comment by Onslau** [comment]
![gtgtg](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/155baa5b-0e88-4974-a873-c1270154828f)
![hh](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/1259c7a5-67d1-495f-95ba-422344976aa9)
![hhh](https://github.com/mathoudebine/turing-smart-screen-python/assets/132712795/94310720-a768-44f1-bfe6-8efc4b8c3000)

**Reply by Onslau** [reply]
[ultimate stat bby Momentum.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11549755/ultimate.stat.bby.Momentum.zip)

**Reply by DJRoby19** [reply]
very nice compliments..
